### PR TITLE
Revert "Revert "Edit comments dialog""

### PR DESCRIPTION
**Pull request by jack-mitt**
This reverts commit 17e8c7f7aba94eabeb11fc3e277f8df320192177.

Refer to #412
I recently updated the buttons, here is a screenshot of the element with the most recent changes:

neweditComments

### CODE REVIEW

**Comment by ash3rz** [comment]
Can you add a screenshot of the updated dialog to the PR?  I'm pretty sure you've updated styles since you came back

File: react-components/src/data/edit/Comment.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import Typography from "@material-ui/core/Typography";
+import ListItem from "@material-ui/core/ListItem";
+import DeleteIcon from "@material-ui/icons/Delete";
+import Fab from "@material-ui/core/Fab";
+import Divider from "@material-ui/core/Divider";
+
+const RETRACTED = "Retracted";
+
+class Comment extends Component {
```
Comment (ash3rz) [inline react-components/src/data/edit/Comment.js]: Looks like all the Comment related stuff under `/data/comments` is still here under `/data/edit` 👀

File: react-components/src/data/comments/EditComments.js [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+                        label={ADD_A_COMMENT}
+                        margin="dense"
+                        multiline
+                        variant="filled"
+                        rows="5"
+                        id="addCommentTextField"
+                    />
+                    <Fab
+                        size="medium"
+                        color={palette.blue}
```
[the remark this answers is not in the document]
Comment (jack-mitt) [reply react-components/src/data/comments/EditComments.js]: new_blue

I personally like the original better, but if you would like me to change it to that and make the plus sign appear in white, I can.

[the remark this answers is not in the document]
Comment (psarando) [reply react-components/src/data/comments/EditComments.js]: I think going with this `primary` color (with a white `+`) would be more consistent with the rest of our dialogs.

For example:
image

File: react-components/src/data/comments/Comment.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            retracted,
+            classes,
+            retractComment,
+        } = this.props;
+
+        return (
+            <>
+                <ListItem id={id}>
+                    <Typography className={classes.commentText}>
+                        On <b>{date}</b> {owner} wrote:
```
Comment (ash3rz) [inline react-components/src/data/comments/Comment.js]: One last bit of text that could be moved over to the `messages` file 😉

**Pull request merged**